frontend/src/app/testResults/uploads/Uploads.tsx

```diff
@@ -25,13 +34,14 @@ const Uploads = () => {
   const [errors, setErrors] = useState<
     Array<FeedbackMessage | undefined | null>
   >([]);
-  const [errorMessageText, setErrorMessageText] = useState(
-    `Please resolve the errors below and upload your edited file. Your file has not been accepted.`
-  );
+  const [errorMessageText, setErrorMessageText] = useState<string | null>(null);
 
   const handleFileChange = async (
     event: React.ChangeEvent<HTMLInputElement>
   ) => {
+    setButtonIsDisabled(true);
+    setFile(undefined);
```

Comment:
this fixes an issue I found while testing:
1.  select a valid file, but don't upload. the Upload button becomes enabled
2. use Change file button to select an empty (or otherwise invalid) file. the Upload button remains enabled
3. click the Upload button - upload occurs, but it uses the previous valid file and will show succeed or error messages based on that file